anagrafica/permessi/funzioni.py

```diff
@@ -309,6 +311,25 @@ def permessi_responsabile_autoparco(sede):
     ]
 
 
+def permessi_delegato_campagne(sede):
+    from donazioni.models import Campagna
+    filtro_sedi = Q(estensione__in=[NAZIONALE, TERRITORIALE])
```

Comment:
Per evitare codice duplicato di questo tipo usiamo un pacchetto chiamato [django-conceptq](https://pypi.python.org/pypi/django-conceptq/0.1.0).

Il modo in cui funziona e' questo:

1. Crea una funzione decorata nella classe che ritorni il filtro adatto, es.:
  ```python
  class Sede(...):
    
    @concept
    def query_puo_avere_campagne(self):
      return Q(estensione__in=[NAZIONALE, TERRITORIALE], ...)
  ```

2. Usalo per ottenere modelli cosi':
  ```python
  sedi = Sede.query_puo_avere_campagne()
  assert isinstance(sedi, QuerySet)
  ```

3. Se ti serve l'oggetto `Q` per mischiarlo ad altro codice, puoi ottenerlo con `.q`, in questo modo:
  ```python
  filtro = Sede.query_puo_avere_campagne().q
  assert isinstance(filtro, Q)
  sedi = Sede.objects.filter(filtro, partita_iva__isnull=True)
  ```